tests/arrays_test.fbs

```diff
@@ -0,0 +1,13 @@
+namespace ArraysTest.Test1;
+
+struct ArrayStruct{
+	a:float;
+	b:[int:15];
```

Comment:
we may want to test a few more types, in a few different alignment scenarios. Like in the current case you can't tell if you're doing padding correctly.